Makefile

```diff
@@ -111,10 +111,11 @@ build-esp32-bin: build-esp32-ota
 		--bin=micro-rdk-server-esp32 \
 		--partition-table=micro-rdk-server/esp32/partitions.csv \
 		--target=xtensa-esp32-espidf \
-		-Zbuild-std=std,panic_abort --release \
+		-Zbuild-std=std,panic_abort \
+		--release \
 		--flash-size=8mb \
 		--merge \
-		target/xtensa-esp32-espidf/micro-rdk-server-esp32.bin
+		target/xtensa-esp32-espidf/release/micro-rdk-server-esp32.bin
```

Comment:
I dislike that we need to name the path to the output directory explicitly here, but I couldn't find a way around it. In any case, the outputs belong in the same directory as the intermediates.